react/src/components/CreateMapModal/CreateMapModal.tsx

```diff
@@ -119,7 +119,15 @@ const CreateMapModal = ({
                     required
                     className={`${styles['input-custom-size']}`}
                   />
-                  <span className={`${styles['hazmapper-suffix']}`}>
+                  <span
+                    className={`${styles['hazmapper-suffix']}`}
+                    style={{
+                      top:
+                        errors.system_file && touched.system_file
+                          ? 'auto'
+                          : '50%',
+                    }}
```

Comment:
**Thoughts.** I tend to avoid `style=` because it "hides" CSS in HTML attributes. I see, basically, why you've done it — you need conditional CSS. We could explore conditional CSS (e.g. via a conditional class), but the adjustment of `top` suggests to me that the root problem is the CSS is not being robust enough to handle layout change gracefully. _I'm not free to test this right now, to investigate. Maybe tomorrow, or Monday._

Reply:
I also avoid CSS in `style` tag because it's excessively specific. [A `style` attribute is as powerful as a Sith lord](https://stuffandnonsense.co.uk/archives/images/css-specificity-wars.png) ([source & context](https://stuffandnonsense.co.uk/archives/css_specificity_wars.html)), when just a Sith apprentice can do the job.

Reply:
I used the conditional class approach in the latest commit